facia-press/app/frontpress/PressedCollectionDeduplication.scala

```diff
@@ -42,11 +42,15 @@ object PressedCollectionDeduplication {
     pCVs.flatMap{ collection => (collection.pressedCollection.curated.take(visibility) ++ collection.pressedCollection.backfill.take(visibility)).map ( pressedContent => pressedContent.header.url ) }
   }
 
+  def makeNewBackfill(collectionV: PressedCollectionVisibility, preceedingCollectionVsDeduplicated: Seq[PressedCollectionVisibility]): List[PressedContent] = {
+    // We want to remove from the current collection' backfilled's PressedCollections those with a header that has already been used
+    val accumulatedHeaderURLsForDeduplication: Seq[String] = getHeaderURLsFromCuratedAndBackfilled(preceedingCollectionVsDeduplicated)
+    collectionV.pressedCollection.backfill.filter( pressedContent => !accumulatedHeaderURLsForDeduplication.contains(pressedContent.header.url) )
+  }
+
   def deduplication(pressedCollections: Seq[PressedCollectionVisibility]): Seq[PressedCollectionVisibility] = {
     pressedCollections.foldLeft[Seq[PressedCollectionVisibility]](Nil) { (accum, collectionV) =>
-      val accumulatedHeaderURLsForDeduplication: Seq[String] = getHeaderURLsFromCuratedAndBackfilled(accum)
-      // We want to remove from the current collection' backfilled's PressedCollections those with a header that has already been used
-      val newBackfill = collectionV.pressedCollection.backfill.filter( pressedContent => !accumulatedHeaderURLsForDeduplication.contains(pressedContent.header.url) )
+      val newBackfill = makeNewBackfill(collectionV, accum)
```

Comment:
Could you pass in only the backfill (i.e. `collectionV.pressedCollection.backfill` instead of `collectionV`) to reduce the context that the `makeNewBackfill`  has? It looks like the only thing we do with `collectionV` in there is call that method chain on it to get the backfill. I think that would make the intent of `makeNewBackfill` a little clearer.

Reply:
+1